libs/client/src/lib.rs

```diff
@@ -396,6 +397,61 @@ impl<R> SolRpcClient<R> {
         RequestBuilder::new(self.clone(), GetBlockRequest::new(params), cycles)
     }
 
+    /// Call `getTokenAccountBalance` on the SOL RPC canister.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use sol_rpc_client::SolRpcClient;
+    /// use sol_rpc_types::{RpcSources, SolanaCluster};
+    /// use solana_pubkey::pubkey;
+    /// use solana_account_decoder_client_types::token::UiTokenAmount;
+    ///
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /// # use sol_rpc_types::{MultiRpcResult, TokenAmount};
+    /// let client = SolRpcClient::builder_for_ic()
+    /// #   .with_mocked_response(MultiRpcResult::Consistent(Ok(TokenAmount {
+    /// #       ui_amount: None,
+    /// #       decimals: 0,
+    /// #       amount: "".to_string(),
+    /// #       ui_amount_string: "".to_string(),
+    /// #    })))
+    ///     .with_rpc_sources(RpcSources::Default(SolanaCluster::Mainnet))
+    ///     .build();
+    ///
+    /// let balance = client
+    ///     .get_token_account_balance(pubkey!("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v"))
```

Comment:
nit: I know it's just a mock, but this address concretely is invalid (not a Token account)
```suggestion
    ///     .get_token_account_balance(pubkey!("3emsAVdmGKERbHjmGfQ6oZ1e35dkf5iYcS6U4CPKFVaa"))
```